### PR TITLE
fix: add routingKey to send-rec

### PR DESCRIPTION
**Pull request by yamii**
S-06459

### CODE REVIEW

File: lib/send-rec/receiver.js [hunk 1]
```diff
@@ -32,8 +37,17 @@ util.inherits( Receiver, Rabbus.Receiver );
...
+			'consume'     : done,
+			'log'         : that.logger,
+			'messageType' : that.messageType,
+			'message'     : message,
+			'emitter'     : that
+		};
+		callback( message, requestStatus( requestOptions ) );
+	} );
 	/*
 	* TODO: Log nack or error emitted
```
Comment (openam) [inline lib/send-rec/receiver.js]: Do we still need this comment?

**Comment by openam** [comment]
I'm good it. Should we do the same thing for publish/subscribe, then update to version 5.0?

**Comment by openam** [comment]
I just realized that we didn't update the `README.md`. We used to `throw err` if there was an error. Do you like logging the error out better?

File: lib/send-rec/receiver.js (hunk 1 above)
Comment (yamii) [inline lib/send-rec/receiver.js]: nope :)

**Pull request merged**